dear_petition/petition/tests/test_data_petition.py

```diff
@@ -0,0 +1,6 @@
+from dear_petition.petition.models import DataPetition
+
+
+def test_data_petition():
+    petition = DataPetition(form_type="sample")
```

Comment:
Is there significance to the `sample` form type? Would `form_type="sample"` be defaulted to `AOCFormCR287` here?

Reply:
Good question. ``sample`` isn't significant as this test only checks for the truthiness of ``data_only``.